bcda/auth/ssas.go

```diff
@@ -171,7 +174,7 @@ func (s SSASPlugin) AuthorizeAccess(tokenString string) error {
 	if !ok {
 		return errors.New("invalid ssas claims")
 	}
-	if _, err = adFromClaims(s.repository, claims); err != nil {
+	if _, err = GetProvider().GetAuthDataFromClaims(claims); err != nil {
 		tknEvent.help = fmt.Sprintf("failed getting AuthData; %s", err.Error())
 		operationFailed(tknEvent)
```

Comment:
out of scope: seems like these `operation<State>` calls should happen one level up, since then it'd be
```
operationStart()
<operation>()
if err {
 operationFailed()
} else {
 operationSuccess()
}